app/routes/user/streaming_steps/credentials.tsx

```diff
@@ -0,0 +1,106 @@
+import { Accordion, Fieldset, Radio } from '@trussworks/react-uswds'
+import { useState } from 'react'
+import { steps, useClient } from '../streaming_steps'
+import ClientCredentialCard from '~/components/ClientCredentialCard'
+import type { ClientCredentialData } from '~/components/ClientCredentialCard'
+import { Link, useLoaderData } from '@remix-run/react'
+import { ClientCredentialVendingMachine } from '../client_credentials.server'
+import type { DataFunctionArgs } from '@remix-run/node'
+import type { AccordionItemProps } from '@trussworks/react-uswds/lib/components/Accordion/Accordion'
+import ClientCredentialForm from '~/components/ClientCredentialForm'
+
+export async function loader({ request }: DataFunctionArgs) {
+  const machine = await ClientCredentialVendingMachine.create(request)
+  const client_credentials = await machine.getClientCredentials()
+  return { client_credentials }
+}
+
+export default function Credentials() {
+  //Data passed between steps
+  const clientData = useClient()
+  // Data loaded for options
+  const { client_credentials } =
+    useLoaderData<Awaited<ReturnType<typeof loader>>>() ?? []
```

Comment:
Why would this ever return null?

Reply:
It shouldn't, I believe that was from testing. Fixing now